src/char.rs

```diff
@@ -23,7 +27,7 @@
 /// done in the `new` function; see its documentation for details.
 ///
 /// The inner `i8` (which is equivalent to `libc::c_char` can be extracted with `.0`, or
-/// by calling `i8::from(my_char)`.
+/// by calling `my_char.to_glib()`.
 #[derive(Debug, Copy, Clone, Eq, PartialEq)]
 pub struct Char(pub i8);
```

Comment:
Sorry for bringing up some C bullshit here, but `char` can be signed or unsigned depending on the platform. On PowerPC, `char` is unsigned for example (and gcc has a `-fsigned-chars` parameter for that very reason). And `gchar` in GLib is equivalent to `char`, so it might be signed or unsigned.

Reply:
Of course nothing in the C standard says that chars must be 8 bits, but let's ignore that for now. All platforms where GLib runs have 8 bit chars.

Reply:
Then maybe completely replace `i8` with `c_char` ?
Then in PowerPC case we have `Char(u8)` and `UChar(u8)` (if it has this definition in `libc`)

Reply:
But then maybe problem with testing.

Reply:
Problem with testing what? But yes, that would make sense. It would at least mirror what happens in GLib then

Reply:
Currently in test just used `65i8` and `65u8`, so "PowerPC" and other will need other tests.

Reply:
@GuillaumeGomez As I remember you prefer `i32` over `c_int` or `gint`.
Can I make reverse here?

Reply:
Added replacement as `fixup!` commit, I do rebase after accepting this way.